onnxruntime/python/tools/transformers/optimizer.py

```diff
@@ -338,7 +341,7 @@ def main():
     if args.input_int32:
         optimizer.change_input_to_int32()
 
-    optimizer.save_model_to_file(args.output)
+    optimizer.save_model_to_file(args.output, args.use_external_data_format)
```

Comment:
Is there any way for optimizer to check whether a model uses external data format or not?

Reply:
When external data format is used, if input and output use the same directory, output external data overwrites input external data, doesn't it?

Reply:
It seems that there's not an API for checking serialized Proto's size;
Yes if we intentionally specify the same input path and output path and the original data is also stored in .data format, then it overwrites. Maybe it's better to give a warning when user does this